modules/chat/src/main/ChatUi.scala

```diff
@@ -64,34 +64,45 @@ final class ChatUi(helpers: Helpers):
       palantir: Boolean = false,
       hostIds: List[UserId] = Nil
   )(using ctx: Context): JsObject =
-    Json
-      .obj(
-        "data" -> Json
+    ctx.kid.yes match
+      case true =>
+        Json
           .obj(
-            "id"         -> chat.id,
-            "name"       -> name,
-            "lines"      -> lines,
-            "resourceId" -> resourceId.value
+            "data" -> false
```

Comment:
this makes the TS types lie
```ts
data: ChatData;
```

Reply:
Done
I put the mandatory fields only with 'sensible' content

Example:
`"chat":{"data":{"id":"","name":"","lines":[],"resourceId":"","loginRequired":true,"restricted":true},"kidMode":true,"noteId":"YHW3eKqs","noteAge":256381}`